exercises/Version-control-basic-pulling-and-pushing-R.md

```diff
@@ -31,18 +31,19 @@ most recent changes from Github.
    ```
 
 2. Click `OK`.
-3. You should see the new function in your repository.
+3. You should see the new lines of code in your `houseelf-analysis.R.
 
    ```
-   get_size_class <- function(ear_length){
-      # Calculate the size class for one or more earth lengths
-      ear_lengths <- ifelse(ear_length > 10, "large", "small")
-      return(ear_lengths)
-   }
+   library(dplyr)
+   
+   data_size_class <-
+     data %>% 
+     rowwise() %>% 
+     transmute(id = id, earlengthcat = get_ear_len_cat(earlength, 10))
    ``` 
 
-4. Write some new code that creates a data frame with information about the
-  individual ID, the earth length class, and the gc-content for each individual.
-5. Save this data frame as a `csv` file using `write.csv()`
-6. Commit the new code and the resulting `csv` file and push the results to
+4. Modifies the code to generate the `data.frame` to include the `gccontent`
```

Comment:
I found the wording confusing. How about "Modify this code to also add a `gccontent`column to the data frame that holds the GC-content for each individual."